shared/chat/conversation/input/index.desktop.js

```diff
@@ -179,8 +205,39 @@ class ConversationInput extends Component<InputProps, State> {
       }
     }
 
+    if (this.props.channelMentionPopupOpen) {
```

Comment:
If you reuse the three counters you could probably also deduplicate most of this function (except for `setChannelMentionPopupOpen`, if that's hard to factor out then :sheep:)